prerequisites/software.rst

```diff
@@ -1,6 +1,60 @@
 Software
 ********
 
+1. Client requirements
+======================
+
+Please note that, while Zammad being a web application, there's some
+requirements for your clients. This ensures that Zammad works as expected.
+
+1.1. Supported Browsers
+-----------------------
+
+.. list-table:: Zammad/Browser version compatibility
+   :header-rows: 1
+   :widths: 20, 30
+
+   * - Firefox 78+
+     - (ESR)
+   * - (Google) Chrome 83+
+     - This also applies for all Chromium based Browsers like Microsoft Edge
+   * - Opera 69+
+     - (based on Chromium 83)
+   * - Microsoft Internet Explorer 11
+     - 
+   * - Safari 11
+     - 
+
+.. note::
+
+   | Zammad heavily uses Javascript which makes it a hard requirement.
+   |
+   | Addons that hook into page content may interfere with Zammads function
+     which is not a bug. 
+   | Google Chromes translation module is known to do
+     odd things to especially state names. Use Zammads internal translations
+     instead.
+
+1.2. Network requirements
+-------------------------
+
+Zammad uses web sockets. Some application firewalls may filter these
+connections. This may lead to decreased browser performance.
+
+There's a fallback to Ajax which  causes a higher application server load
+and thus should be avoided.
+
+   .. note::
+
+      Core workflows comes with an `Ajax Mode`_ which can be activated
```

Comment:
Do people understand at this point that "Core workflows" refers to a Zammad feature? Maybe put it in quotes or rephrase it like The "Core Workflows" feature of Zammad comes with...